#163415213: implement the delete confirmation modal

Chiazokam opened the pull request:
#### What does this PR do?
This PR modifies designs a modal to give the user a confirmation message on delete

#### Description of Task to be completed?
- design the delete confirmation modal

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ft-ui-delete-modal-163415213 `
- open the index.html file on the browser and navigate to the desired pages

#### What are the relevant pivotal tracker stories?
[#163415213](https://www.pivotaltracker.com/story/show/163415213)

### Images

![image](https://user-images.githubusercontent.com/26940294/51595085-3155b780-1ef6-11e9-8830-e66d8bbd2be5.png)

Chiazokam committed:
feat (frontend): include column title in the view pages

- include column title in the view pages
- reduce the height of the party logo in mobile
- reposition the submit button on the vote page
- modify the hover color of text on the vote submit button
- make pages mobile responsive

[Delivers #163377772]

Chiazokam committed:
feat (frontend): include column title in the view pages

- include column title in the view pages
- reduce the height of the party logo in mobile
- reposition the submit button on the vote page
- modify the hover color of text on the vote submit button
- make pages mobile responsive
-  make the the row titles bold in the view pages
- remove texts in strip on view messages
- reduse the font size of the office column
- remove shadows in column titles
- make titles bold
- centralize vote submit button
- change column title on mobile view pages

[Delivers #163377772]

Chiazokam committed:
feat (frontend): make the the row titles bold in the view pages

-  make the the row titles bold in the view pages
- change the color of the title header and reduce height
- reduce spaces in the form
- make candidate and party display responsive
- link logo to the home page

[Delivers #163377772]

Chiazokam committed:
feat (frontend): make the the row titles bold in the view pages

-  make the the row titles bold in the view pages

[Delivers #163377772]

Chiazokam committed:
Merge branch 'ft-ui-column-title-163377772' of https://github.com/Chiazokam/Politico into ft-ui-column-title-163377772

Chiazokam committed:
Merge branch 'develop' into ft-ui-column-title-163377772

Chiazokam committed:
feat (frontend): modify the modal

- reduce the width of the modal box
- close modal on clicking the screen
- centralize modals

[Delivers #163411569]

Chiazokam committed:
feat (frontend): modify the modal

- reduce the width of the modal box
- close modal on clicking the screen
- centralize modals

[Delivers #163411569]

Chiazokam committed:
feat (frontend): implement the delete confirmation modal

- design the delete confirmation modal

[Delivers #163415213]

Chiazokam committed:
feat (frontend): implement the delete confirmation modal

- design the delete confirmation modal

[Delivers #163415213]

JCanaks commented:
Nice work. Just reduce the width of the modal and centralize the modal to the center of the page and this will be good to go

Chiazokam commented:
> Nice work. Just reduce the width of the modal and centralize the modal to the center of the page and this will be good to go

Thanks.   Most of the changes (Eg the width of all modals) has been implemented on one branch. 
Is it ok to merge so that the changes will reflect on other branches or do I work on the features on each branch?

Chiazokam committed:
Merge branch 'ft-ui-column-title-163377772' of https://github.com/Chiazokam/Politico into ft-ui-column-title-163377772

Chiazokam committed:
feat (frontend): fix conflict on parent branch

	- include column title in the view pages
    - reduce the height of the party logo in mobile
    - reposition the submit button on the vote page
    - modify the hover color of text on the vote submit button
    - make pages mobile responsive
    -  make the the row titles bold in the view pages
    - remove texts in strip on view messages
    - reduse the font size of the office column
    - remove shadows in column titles
    - make titles bold
    - centralize vote submit button
    - change column title on mobile view pages

[Delivers #163440686]

Chiazokam committed:
feat (frontend): fix conflict on parent branch

	- include column title in the view pages
    - reduce the height of the party logo in mobile
    - reposition the submit button on the vote page
    - modify the hover color of text on the vote submit button
    - make pages mobile responsive
    -  make the the row titles bold in the view pages
    - remove texts in strip on view messages
    - reduse the font size of the office column
    - remove shadows in column titles
    - make titles bold
    - centralize vote submit button
    - change column title on mobile view pages

[Delivers #163440686]

Chiazokam committed:
Merge branch 'ft-ui-header-title-163440686' of https://github.com/Chiazokam/Politico into ft-ui-header-title-163440686

Chiazokam committed:
Merge pull request #23 from Chiazokam/ft-ui-header-title-163440686

#163440686 fix column title in the view pages

Chiazokam committed:
Merge branch 'ft-ui-view-modal-163411569' of https://github.com/Chiazokam/Politico into ft-ui-view-modal-163411569

Chiazokam committed:
Merge pull request #21 from Chiazokam/ft-ui-view-modal-163411569

#163411569 modify the modal

Chiazokam committed:
Merge branch 'ft-ui-delete-modal-163415213' of https://github.com/Chiazokam/Politico into ft-ui-delete-modal-163415213
